app/views/books/index.html.haml

```diff
@@ -7,10 +7,20 @@
   %li
     .btn.btn-outline-danger.btn-sm
       = link_to "Add New Subject", trestle.new_subjects_admin_path
+
 - if @books.blank?
   %p There are not any books currently in the system.
+
 - else
-  %p These are the current books in our system
+  %h3 Search Books
+
+  = search_form_for @q do |f|
+    // Search if the title field contains query
+    %p Title:
```

Comment:
For this let's leverage a `label`. A label in a form provides accessibility improvements for the user and you should get in the practice of using them!
```suggestion
    = f.label :title
```